src/report_error.ml

```diff
@@ -40,11 +40,16 @@ let get_printer = function
           (User_message.pp { msg with loc = None; hints = [] })
     }
   | Code_error.E t ->
+    let maybe_pp_loc fmt =
+      Option.iter ~f:(Format.fprintf fmt "@.%a" Loc.print) in
     let pp = fun ppf ->
       Format.fprintf ppf "@{<error>Internal error, please report upstream \
                           including the contents of _build/log.@}\n\
-                          Description:%a\n"
-        Pp.render_ignore_tags (Dyn.pp (Code_error.to_dyn t))
+                          Description:%a%a\n"
```

Comment:
Did you look at an output example? I don't know if the absence of whitespace is right.

Reply:
It should be fine as I am making sure the printer itself includes the whitespace.

It's a bit hard to add a test case since these exceptions are a sign of a bug in dune.